fastify.js

```diff
@@ -124,90 +126,111 @@ function build (options) {
   return fastify
 
   function fastify (req, res) {
-    logger(req, res)
+    req.id = genReqId(req)
+    req.log = res.log = logger.child({ req: req })
+    res[startTime] = res[startTime] || Date.now()
```

Comment:
Why?
This should be in a plugin.
IMHO tThere's no reason to calculate the `responseTime` if explicitly done

Reply:
we can use `process.hrtime()`. I think it would be faster (but we should bench).
I would remove the `startTime` symbol, as we have internalized the logging system.